app/views/analytics/index.html.erb

```diff
@@ -147,26 +147,59 @@
     </div>
   </div><!-- row   -->
 
-  <!-- Comments List-->
   <div class="row">
-    <div class="col-sm-12">
-      <div class="chart-wrapper">
-        <div class="chart-title background-orange">
-          Recent Comments
-        </div>
-        <ul class="scroller">
-          <% @most_recent_comments.each do |comment| %>
-            <li>
-              <blockquote class="blockquote">
-                <p><%= link_to_case_title(comment.commentable, 25) %> -- <%= comment_created_at(comment) %></p>
-                <p><%= link_to_comment(comment, 100) %></p>
-                <small><%= link_to_comment_user(comment) %></small>
-              </blockquote>
-            </li>
-          <% end %>
-        </ul>
+    <!-- Editors List-->
+      <div class="col-sm-4">
+        <div class="chart-wrapper">
+          <div class="chart-title background-orange">
+            Most Recent Editors - Last 30 days <%= image_tag('help_icon.png', data: {:toggle => 'popover', :container => 'body', :trigger => 'hover',:placement => "bottom", :title => "Active Editors", :content => "This metric shows the number of cases added or updated by editors in the past 30 days."})%>
+          </div>
+          <div class="chart-stage">
+           <table class='table leader-list'>
+             <tr>
+              <th>Editor</th>
+              <th>Updates</th>
+             </tr>
+
+             <% case_updaters_count(10).each do |who| %>
+               <% user_id = who[0].to_i %>
+               <% user_object = who[1] %>
+               <tr>
+                 <% unless user_id == 0 %>
+                   <td><%= @user = User.find(user_id).name %></td>
+                   <td><%= link_to user_object, @user %></td>
+                 <% else %>
+                   <td> Anonymous </td>
```

Comment:
How is this branch even possible if only signed in users can edit cases?

Reply:
Cases are edited via the admin panel, rails console and rake tasks, where current_user is nil.